e2e_samples/temperature_events/LoadTesting/LoadTestCheckResult.ps1

```diff
@@ -0,0 +1,48 @@
+<#
+This script fetches the Incoming Messages and Outgoing Messages Metrics for the eventhub
+where the IoT Simulator sends the load to. If the total num of Incoming Messages <= Outgoing Messages, 
+we let this automated load test pass. We suggest that you look into the metrics through portal manually 
+again to confirm or gain more insights about whether the system is behaving as expected.
+#>
+param (
+    [Parameter(Mandatory=$true)][string]$evhSubscriptionId,
+    [Parameter(Mandatory=$true)][string]$evhResourceGroup,
+    [Parameter(Mandatory=$true)][string]$evhNamespace,
+    [Parameter(Mandatory=$true)][string]$evhName
+ )
+
+$evhNamespaceResourceId="/subscriptions/$evhSubscriptionId/resourceGroups/$evhResourceGroup/providers/Microsoft.EventHub/namespaces/$evhNamespace"
+$currentTime = Get-Date -Format "o"
+
+# Get the total number of incoming messages.
+$ingress_metric = az monitor metrics list `
+--resource $evhNamespaceResourceId `
+--metrics "IncomingMessages" `
+--filter "EntityName eq '$evhName' " `
+--start-time $env:LOADTESTSTARTTIME `
+--end-time $currentTime `
+--interval 24h | ConvertFrom-Json
+
+$ingress_num = $ingress_metric.value[0].timeseries[0].data[0].total
+
+# Get the total number of outgoing messages.
+$egress_metric = az monitor metrics list `
+--resource $evhNamespaceResourceId `
+--metrics "OutgoingMessages" `
+--filter "EntityName eq '$evhName' " `
+--start-time $env:LOADTESTSTARTTIME `
+--end-time $currentTime `
+--interval 24h | ConvertFrom-Json
```

Comment:
`24h` load test maybe on the extreme side. Also lets make this a parameter

Reply:
Just to clarify, 24h is not for running the load test for 24 hours :D it is the unit of time to calculate the total number of events within the specified start time and end time. I chose 24 hr to ensure that I only get one total number for the entire load testing period instead of several numbers per minute or hour aggregation.
But yes I can make this into a parameter as well :)